invisible_cities/cities/dorothea.py

```diff
@@ -57,15 +60,33 @@
 from .  components import compute_xy_position
 from .  components import build_pointlike_event  as build_pointlike_event_
 
+from typing import Optional
 
-@city
-def dorothea(files_in, file_out, compression, event_range, print_mod, detector_db, run_number,
-             drift_v,
-             s1_nmin, s1_nmax, s1_emin, s1_emax, s1_wmin, s1_wmax, s1_hmin, s1_hmax, s1_ethr,
-             s2_nmin, s2_nmax, s2_emin, s2_emax, s2_wmin, s2_wmax, s2_hmin, s2_hmax, s2_ethr, s2_nsipmmin, s2_nsipmmax,
-             global_reco_params=dict(),
-             include_mc = False):
 
+@city
+def dorothea( files_in          : OneOrManyFiles
+            , file_out          : str
+            , compression       : str
+            , event_range       : EventRangeType
+            , print_mod         : int
+            , detector_db       : str
+            , run_number        : int
+            , drift_v           : float
+            , s1_nmin           :   int, s1_nmax          :   int
+            , s1_emin           : float, s1_emax          : float
+            , s1_wmin           : float, s1_wmax          : float
+            , s1_hmin           : float, s1_hmax          : float
+            , s1_ethr           : float
+            , s2_nmin           :   int, s2_nmax          :   int
+            , s2_emin           : float, s2_emax          : float
+            , s2_wmin           : float, s2_wmax          : float
+            , s2_hmin           : float, s2_hmax          : float
```

Comment:
I find the spatial proximity of the type of the earlier parameter to the name of the next parameter, a bit confusing to read. That is to say, in something like
```python
, s1_nmin          : float, s1_emax          :float
```
the natural inclination is to see the `float, s1_emax` as belonging together.

Reply:
I'm planning a PR applying a more uniform style everywhere, all these cosmetic changes will fit in there naturally.